### DISCUSSION TITLE
Is v7 version released on https://get.pnpm.io?

### DISCUSSION THREAD

**Discussion by theolavaux** [pnpm/pnpm#4871] [Q&A]
Tried to install v7 from this link https://get.pnpm.io/v7.1.js in CI but it seems not deployed yet, is there a place to see live versions link ?

**Comment by zkochan** [comment]
It didn't matter what script is there. You can do

```
curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm@latest-7
```

**Comment by mcmxcdev** [comment]
You can always see the latest published pnpm versions on https://github.com/pnpm/pnpm/releases or https://github.com/pnpm/pnpm/tags
